.webpack/webpack.common.mjs

```diff
@@ -52,7 +52,37 @@ const commonConfig = {
             export: 'default',
             name: 'openmctYamcs'
         }
-    }
+    },
+    devServer: {
+        compress: true,
+        port: 9000,
+        static: [
+          {
+            directory: path.resolve(projectRootDir, './example'),
+          },
+          {
+            directory: path.resolve(projectRootDir, './node_modules/openmct/dist'),
+            publicPath: '/node_modules/openmct/dist'
+          },
+        ],
+        proxy: [
+          {
+            context: ["/yamcs-proxy/"],
+            target: "http://0.0.0.0:8090/",
```

Comment:
Have some mercy for those of us who are targeted by enterprise port scanners, please.